extensions/http-curl/tests/C2VerifyHeartbeatAndStop.cpp

```diff
@@ -162,12 +162,14 @@ class VerifyC2Heartbeat : public CoapIntegrationBase {
     std::string url = "";
     inv->getProperty(minifi::processors::InvokeHTTP::URL.getName(), url);
 
+    std::string c2_url = std::string("http") + (isSecure ? "s" : "") + "://localhost:" + getWebPort() + "/api/heartbeat" ;
```

Comment:
Is this new functionality? I don't see https in this case before.

Reply:
Just added this as the class has an "isSecure" member, but it wasn't used. 
Made it consistent.